ice40/tests/blink/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
+set(BOARDS iceblink40-lp1k icestick)
+
+if (NOT DEFINED ENV{CI} OR NOT $ENV{CI})
+set(BOARDS ${BOARDS} hx8k-b-evn icevision tinyfpga-b2 tinyfpga-bx)
+endif (NOT DEFINED ENV{CI} OR NOT $ENV{CI})
```

Comment:
I think list(APPEND BOARDS is more idiomatic, but this works too

Reply:
Done